src/shared/path-lookup.c

```diff
@@ -421,6 +421,34 @@ static int acquire_control_dirs(UnitFileScope scope, char **persistent, char **r
         return 0;
 }
 
+static int acquire_attached_dirs(
+                UnitFileScope scope,
+                char **ret_persistent,
+                char **ret_runtime) {
+
+        _cleanup_free_ char *a = NULL, *b = NULL;
+
+        assert(ret_persistent);
+        assert(ret_runtime);
+
+        /* Portable services are not available to regular users for now. */
+        if (scope != UNIT_FILE_SYSTEM)
+                return -EOPNOTSUPP;
+
+        a = strdup("/etc/systemd/system.attached");
+        if (!a)
```

Comment:
I think `/etc/systemd/attached/` would be better as it is shorter without loosing any information (I don't think there will ever be a `user.attached` directory).

Reply:
we already have /etc/systemd/system.control + /etc/systemd/user.control, hence this follows an existing naming scheme.

Also, eventually I'd like to open this up for user services too, hence let's prepare for that...